documentation/User-Documentation/Notifications.md

```diff
@@ -1,148 +1,212 @@
+| Language Changed | language_changed, *info* |
+| Mobs to Kill | mobs_to_kill, *info* |
+| Money Given | money_given, *info* |
+| Money Taken | money_taken, *info* |
+| Quest Cancelled | quest_cancelled, *info* |
+| Items Given | items_given, *info* |
+| New Journal Entry | new_journal_entry, *info* |
+| Items Taken | items_taken, *info* |
+| Blocks to Break | blocks_to_break, *info* |
+| Blocks to Place | blocks_to_place, *info* |
+| Animals to Breed | animals_to_breed, *info* |
+| Mobs to click | mobs_to_click, *info* |
+| Fish to catch | fish_to_catch, *info* |
+| Players to kill | players_to_kill, *info* |
+| Potions to brew | potions_to_brew, *info* |
+| Points given | point_given, *info* |
+| Points taken | point_taken, *info* |
+| Points multiplied | point_multiplied, *info* |
+| Sheep to shear | sheep_to_shear, *info* |
+
+## Adding custom Notifications using events
+
+A truly custom notification can be sent using the `notify` event. You can either directly define all Notify IO options like so:
+```YAML
+myEvent: "notify This is a custom message. io:bossbar barColor:red sound:block.anvil.use"
+```
 
-### Suppress
-Does not output anything. Can be used to stop notifications.
+Or you can use categories from the *custom.yml* like so:
+```YAML
+myEvent: "notify This is a custom message! category:info"
+```
+
+You can also override category settings:
+```YAML
+myEvent: "notify Another messsage! category:info io:advancement frame:challenge"
+```
 
+These custom notifications also allow custom categories. You can go wild here:
+```YAML
+notifications:
+  money: 
+    io: bossbar
+    icon: gold_ingot  
+```
+   
+!!! warning 
+    All colons (`:`) in your notification messages need to be escaped with one backslash (`\`) when using single quotes
+    (`''`) and with two backslashes (`\\`) when using double quotes (`""`). Example:  `'Peter:Heya %player%!' -> 'Peter\:Heya %player%!'` `"Peter:Heya %player%!" -> "Peter\\:Heya %player%!"`
+     
+    
+
+## Notify IO's
 ### Chat
-Writes the notification to the players chat.
+Writes the notification in the players chat.
+
+??? info "Preview"
+    ![chat image](../media/content/User-Documentation/Notifications/chat.png)
 
 | Option | Description |
 |--------|-------------|
-| sound | Sound to play. If blank, no sound. Can be from [here](https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/Sound.html) or the name of a sound including from a custom resource. |
+| sound | Sound to play. If blank, no sound. Either <a href="https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/Sound.html" target="_blank"> vanilla Minecraft sounds</a> or the name of a sound from a resource pack. |
 
 ### Advancement
-Shows the notification using an achievement popup.
+Shows the notification using an achievement popup. Unfortunately Minecraft does play a sound here that cannot be disabled 
+unless you remove it from your ressource pack. You can still add your own additional sound to this notification though.
```

Comment:
I asked my self, if the original sound is still played, but not so loud as the new sound.

Reply:
formulierung